tests/scalers/prometheus.test.ts

```diff
@@ -153,6 +160,7 @@ spec:
   cooldownPeriod:  10
   triggers:
   - type: prometheus
+    metricType: AverageValue
```

Comment:
Why do we need to set this? This should be a default value, so I prefer if we omit it from the test.